testing/cypress/fixtures/axe-pages.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
this file will be used for both local and pipeline set up... this will help us in the long run to potentially use multiple envs